tests/unit/amo/test_store.js

```diff
@@ -12,115 +16,78 @@ describe(__filename, () => {
 
   it('includes logging middleware on the server while in development', () => {
-    const _applyMiddleware = sinon.stub();
-    const _minimalReduxLogger = sinon.stub();
+    const _minimalReduxLogger = 'This is a logger';
 
     expect(
-      typeof middleware({
-        _applyMiddleware,
+      middleware({
         _config: configForDev(true, { server: true }),
         _minimalReduxLogger,
-      }),
-    ).toBe('function');
-
-    sinon.assert.calledWith(
-      _applyMiddleware,
-      sinon.match((...args) => {
-        return args.includes(_minimalReduxLogger);
-      }),
-    );
+      })[0],
+    ).toEqual(_minimalReduxLogger);
   });
 
   it('does not apply middleware if not in development', () => {
-    const _createLogger = sinon.stub();
+    const _createLogger = jest.fn();
     expect(
-      typeof middleware({
+      middleware({
         _config: configForDev(false),
         _createLogger,
       }),
-    ).toBe('function');
-    sinon.assert.notCalled(_createLogger);
-  });
-
-  it('handles a falsey window while on the server', () => {
-    const _createLogger = sinon.stub();
-    const _window = null;
-    expect(
-      typeof middleware({
-        _config: configForDev(true),
-        _createLogger,
-        _window,
-      }),
-    ).toBe('function');
-    sinon.assert.called(_createLogger);
+    ).toHaveLength(0);
+    expect(_createLogger).not.toHaveBeenCalled();
   });
 
-  it('does not create a logger for the server', () => {
-    const _createLogger = sinon.stub();
+  it('does not create a logger via createLogger for the server', () => {
+    const _createLogger = jest.fn();
     expect(
       typeof middleware({
         _config: configForDev(true, { server: true }),
         _createLogger,
-      }),
+      })[0],
     ).toBe('function');
-    sinon.assert.notCalled(_createLogger);
+    expect(_createLogger).not.toHaveBeenCalled();
   });
 
-  it('uses a placeholder store enhancer when devtools is not available', () => {
-    const _window = {}; // __REDUX_DEVTOOLS_EXTENSION__() is undefined
+  it('includes devtools when config enables it', () => {
     const _config = getFakeConfig({ enableDevTools: true });
 
-    const enhancer = middleware({ _config, _window });
-
-    expect(typeof enhancer).toBe('function');
-
-    const fakeCreatestore = () => {};
-    expect(typeof enhancer(fakeCreatestore)).toBe('function');
-  });
-
-  it('adds the devtools store enhancer when config enables it', () => {
-    const _window = {
-      __REDUX_DEVTOOLS_EXTENSION__: sinon.spy(),
-    };
-    const _config = getFakeConfig({ enableDevTools: true });
+    expect(includeDevTools({ _config })).toEqual(true);
 
-    expect(typeof middleware({ _config, _window })).toBe('function');
-    sinon.assert.called(_window.__REDUX_DEVTOOLS_EXTENSION__);
+    // TODO: This doesn't get called, and probably doesn't even need to be checked,
+    // because of RTK.
+    // expect(_window.__REDUX_DEVTOOLS_EXTENSION__).toHaveBeenCalled();
```

Comment:
was this `TODO` intended to be finished before the patch was finished, or at a later date?

Reply:
Thanks, my intention was to remove that stuff. I'll do that now.